### ISSUE TITLE
Semicolon within string in an SQL statement confuses end-of-statement detection

### ISSUE THREAD

**Issue by rohanc** [TablePlus/TablePlus-Windows#70]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
Postgres 9.6.8
2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
1.2 (72)
3. The steps to reproduce this issue:

Open a new SQL query tab.
Paste the following text:

```sql
update event set await_message='<img class=''ssbanner'' src=''images/ser/ssbanner.jpeg'' style=''width: 930px;'' alt=''School application page'' /><p style=''color: red;''><b>testing</b></p>' where staffkey=123;
```

Click the mouse somewhere within the word "update". TablePlus highlights only half of the statement, ending after "930px;"
Instead it ought to highlight the entire statement after "123;". 

In other words, only semicolons outside of string delimiters should be used to determine the end of an SQL statement.

**Comment by huyphams** [comment]
Hey, I fixed this issue on the build 82, let me know if you're still seeing it.

**Comment by rohanc** [comment]
Yes, looks better now, thanks.
